tests/fv3_conf/gfs_v16_run.IN

```diff
@@ -19,6 +19,15 @@ cp    @[INPUTDATA_ROOT]/FV3_input_data/*_table .
 cp    @[INPUTDATA_ROOT]/FV3_input_data/diag_table_gfdlmp diag_table
 cp    @[INPUTDATA_ROOT]/FV3_input_data_gsd/field_table_suite2 field_table
 cp    @[INPUTDATA_ROOT]/FV3_input_data/*configure .
+cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.*nc .
+for n in 01 02 03 04 05 06 07 08 09 10 11 12; do
+ln -sf merra2.aerclim.2003-2014.m${n}.nc  aeroclim.m${n}.nc
```

Comment:
Instead of this, you could copy directly from input directory, like:

`cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m${n}.nc aeroclim.m${n}.nc`

or even better link directly from input directory. Fewer files in run directory.